content/docs/start/data-and-model-access.md

```diff
@@ -2,29 +2,29 @@
 title: 'Get Started: Data and Model Access'
 ---
 
-# Get Started: Data and Model Access
+# Data and Model Access
 
-Okay, we've learned how to _track_ data and models with DVC, and how to commit
-their versions to Git. The next questions are: How can we _use_ these artifacts
-outside of the project? How do I download a model to deploy it? How to download
+We've learned how to _track_ data and models with DVC, and how to commit their
```

Comment:
I did my best to not alter the tone of the docs (which I would describe as absolutely abhorrent) but couldn't bring myself to let this one go. All of my other changes though are grammatical corrections or at least increase precision (leaving less room for misinterpretation).

Reply:
The original author here was @shcheklein if I remember correctly but of course others reviewed this at the time and passed the process. I think we need to trust our own processes, so far we have good results (esp. the GS is very popular and never had a complaint on tone, well we have one now).

But any improvement suggestions that can be explained are def. welcome. TBH it's still unclear to me what the reason is for this change. We have playful language here and there throughout the website. Hard to tell whether it has a negative impact but I doubt it. Could be wrong

Reply:
BTW the change can def stay now since it's done. I'm just saying that we should try to focus on more impactful things that aren't subjective in nature, esp. for existing writing tone and style. Thanks

Reply:
marking as resolved; continuing discussion about style below in other comments.